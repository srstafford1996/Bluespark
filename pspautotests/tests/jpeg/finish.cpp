#include <common.h>
#include <pspjpeg.h>
#include <psputility.h>

extern "C" int main(int argc, char *argv[]) {
	checkpointNext("Modules:");
	checkpoint("  LoadAv: %08x", sceUtilityLoadAvModule(PSP_AV_MODULE_AVCODEC));
	checkpoint("  UnloadAv: %08x", sceUtilityUnloadAvModule(PSP_AV_MODULE_AVCODEC));
	checkpoint("  Load: %08x", sceUtilityLoadModule(PSP_MODULE_AV_AVCODEC));

	checkpointNext("Sequence:");
	checkpoint("  Init: %08x", sceJpegInitMJpeg());
	checkpoint("  Finish: %08x", sceJpegFinishMJpeg());
	checkpoint("  Twice: %08x", sceJpegFinishMJpeg());

	checkpointNext("In use:");
	sceJpegInitMJpeg();
	checkpoint("  Create: %08x", sceJpegCreateMJpeg(480, 272));
	checkpoint("  Finish in use: %08x", sceJpegFinishMJpeg());
	checkpoint("  Delete: %08x", sceJpegDeleteMJpeg());
	checkpoint("  Finish deleted: %08x", sceJpegFinishMJpeg());

	return 0;
}