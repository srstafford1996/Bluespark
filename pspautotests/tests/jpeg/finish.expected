[x] Modules:
[r]   LoadAv: 00000000
[r]   UnloadAv: 00000000
[r]   Load: 00000000

[x] Sequence:
[r]   Init: 00000000
[r]   Finish: 00000000
[x]   Twice: 80650039

[x] In use:
[r]   Create: 00000000
[x]   Finish in use: 80650039
[x]   Delete: 00000000
[r]   Finish deleted: 00000000
